tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -30,21 +30,21 @@
     'x86_64': {
         '2vcpu_256mb.json': {
             'FULL':  180,
-            'DIFF':  50,
```

Comment:
What happened here (as well as the other benchmarks)? Does the ramdisk impact the performance tests this severely?

Reply:
Yes, well, not the ramdisk, but the flushing operation on all disks and all dirtied memory. To get an idea of the cost, without ramdisk the operation takes 2300ms, and under 50ms with ramdisk.

The flush still implies FS operations which add a bit of latency. I am not at all concerned about this value.